package.json

```diff
@@ -31,7 +31,7 @@
     "@types/bluebird": "^3.5.26",
     "@types/depcheck": "^0.6.0",
     "@types/glob": "^5.0.35",
-    "@types/node": "^8.10.45",
+    "@types/node": "^10.14.10",
```

Comment:
Why was this necessary?  Mainly asking because it's potentially a breaking change

Reply:
should be possible to revert - I needed this because the original implementation was using `fs.readDir` variation that was returning `Dirent[]`, and it was introduced in later versions.

But since I rewrote the checker to fit the rest of the lint code, it's not longer necessary.